engineio/async/threading.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import absolute_import
+
+import threading
+
+try:
+    import queue
+except ImportError:  # pragma: no cover
+    import Queue as queue
+Queue = queue.Queue
+QueueEmpty = queue.Empty
+has_websocket = False
+
+
+def thread(target):
+    return threading.Thread(target=target)
+
+
+def wrap_websocket(app):
+    raise RuntimeError("Websocket support is not available")
```

Comment:
I think it may be a bit more clear to make `wrap_websocket()` return None to indicate this transport isn't available. That way you don't need to have `has_websocket`.